component/core/src/main/java/org/exoplatform/social/core/listeners/SocialMembershipListenerImpl.java

```diff
@@ -118,6 +118,10 @@ public void postSave(Membership m, boolean isNew) throws Exception {
             return;
           }
           spaceService.addMember(space, userName);
+        } else if (SpaceUtils.REDACTOR.equalsIgnoreCase(m.getMembershipType())) {
+          if (spaceService.isRedactor(space, userName))
+            return;
+          spaceService.addRedactor(space, userName);
```

Comment:
```suggestion
        } else if (SpaceUtils.REDACTOR.equalsIgnoreCase(m.getMembershipType())) {
          if (spaceService.isRedactor(space, userName)) {
            return;
          }    
          spaceService.addRedactor(space, userName);
```

Reply:
does this make difference ?

Reply:
No change, but I suggest being in harmony with the above implementation.